Tools/SelectionTool/SelectionTool.cs

```diff
@@ -35,14 +35,18 @@ sealed class SelectionTool : MonoBehaviour, ITool, IUsesRayOrigin, IUsesRaycastR
 		Color m_MultiselectRayColor;
 		bool m_MultiSelect;
 
+		readonly Dictionary<string, List<VRInputDevice.VRControl>> m_Controls = new Dictionary<string, List<VRInputDevice.VRControl>>();
```

Comment:
we might want to consider some kind of container class for this - kind of a big function signature to drop on everyone

Reply:
Hm. So it would just be a container that wraps the big fat dictionary type? I guess that's not a bad idea, especially since we could use the same kind of thing to cut down on that really really long string of generics in TwoHandedProxyBase